components/SubTrackDetails/SubTrackDetailsExample.jsx

```diff
@@ -0,0 +1,23 @@
+import SubTrackDetails from './SubTrackDetails'
+import React from 'react'
+
+/** Temporarly div Style can be replaced by tool tip componenet **/
+const divStyle = {
+  backgroundColor:'#47474F',
+  padding:'20px',
+  width:'320px'
+}
+
+const challengeDetails = {
+  name : 'First To Finish (F2F)',
```

Comment:
The subtrack name (First To Finish) passed in will be separate from the code (F2F). These should be separate props.